### PR TITLE
Fix the error when delete some images

### LINKED ISSUE #9603: Unorganized error warning --test   immutable
I have created an immutable-tag ,and push images,  when delete an image that has contact with immutable tag , it can not delete  this image, it is right ,but  the error message is wrong

image

### LINKED ISSUE #9604: delete images error
When I delete multiple images, which contains images that cannot be deleted, the delete display fails, but the background has been deleted. The foreground is not refreshed.
image

### PR DESCRIPTION
**Pull request by jwangyangls**
Signed-off-by: Yogi_Wang <yawang@vmware.com>
fix #9604  fix #9603

### CODE REVIEW

**Comment by zhoumeina** [comment]
image

**Comment by zhoumeina** [comment]
image

**Comment by zhoumeina** [comment]
this text is not vertical center. And text should be 13px.

File: src/portal/lib/src/shared/shared.utils.ts [hunk 1]
```diff
@@ -22,19 +22,43 @@ export const errorHandler = function (error: any): string {
...
     try {
         return JSON.parse(error.error).message;
     } catch (err) { }
+    // Not a standard error return Basically not used cover unknown error
     if (typeof error.error === "string") {
         return error.error;
     }
     if (error.error && error.error.message) {
-        return error.error.message;
+        // handle docker client response error
```
Comment (zhoumeina) [inline src/portal/lib/src/shared/shared.utils.ts]: Add unit test for this logic.

**Comment by AllForNothing** [comment]
In the future , we need to take more care of  batch operation

**Pull request merged**